assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-shopping-button-block/constants.tsx

```diff
@@ -0,0 +1,9 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const defaultShoppingButtonLabel = __(
```

Comment:
Maybe `defaultStartShoppingButtonLabel` would be better. It took me a moment to figure out what `defaultShoppingButtonLabel` meant when I looked at it elsewhere in the code

Reply:
I see what you mean. I went ahead and applied the following changes:

- `defaultShoppingButtonLabel` renamed to `defaultStartShoppingButtonLabel` 
- `shoppingButtonLabel` renamed to `startShoppingButtonLabel`